pkgs/development/python-modules/openapi-python-client/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  poetry-core,
+  pyyaml,
+  typer,
+  autoflake,
+  httpx,
+  python-dateutil,
+  jinja2,
+  black,
+  pydantic,
+  attrs,
+  isort,
+  shellingham,
+}:
+
+buildPythonPackage rec {
+  pname = "openapi_python_client";
+  version = "0.14.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-R/mG9DSUogNSHqAeBfdxL2HUvkHNu9kMJuFcu6G7kLs=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+    pyyaml
+    typer
+    autoflake
+    httpx
+    python-dateutil
+    jinja2
+    black
+    pydantic
+    attrs
+    isort
```

Comment:
```suggestion
    pydantic
    attrs
```
we don't do linting

Reply:
If I remove `pydantic`, `black` and `isort`, tests are failing. Removing `attrs` only is ok.

Reply:
How to ignore the tests? I can't even run them since `pydantic` is required...

![image](https://github.com/NixOS/nixpkgs/assets/252042/3d96842e-3b8c-4f2c-87b0-938fd6eda874)

Reply:
Fixed.